spec.html

```diff
@@ -8538,8 +8528,7 @@ <h1>AsyncGeneratorFunctionCreate ( _kind_, _ParameterList_, _Body_, _Scope_ )</h
       <p>The abstract operation AsyncGeneratorFunctionCreate requires the arguments: _kind_ which is one of (~Normal~, ~Method~), a parameter list Parse Node specified by _ParameterList_, a body Parse Node specified by _Body_, a Lexical Environment specified by _Scope_. AsyncGeneratorFunctionCreate performs the following steps:</p>
       <emu-alg>
         1. Let _functionPrototype_ be %AsyncGenerator%.
-        1. Let _F_ be ! FunctionAllocate(_functionPrototype_, ~generator~).
-        1. Return ! FunctionInitialize(_F_, _kind_, _ParameterList_, _Body_, _Scope_).
+        1. Return ! OrdinaryFunctionCreate(_functionPrototype_, _ParameterList_, _Body_, ~generator~, _kind_, _Scope_).
```

Comment:
Existing bug: `~generator~` seems wrong here. Shouldn't this be `~asyncGenerator~`?

Reply:
If so let's fix that in a separate commit (in this PR would be preferred, ideally fixing it before the editorial change)

Reply:
If #1546 lands before this one it makes my comment non-applicable, since function kind will be gone.

Reply:
> Shouldn't this be `~asyncGenerator~`?

That was my question in issue #1544 (modulo the changes of PR #1725), which then resulted in PR #1546.

Reply:
Resolved by merging #1546, which eliminates the parameter in question.